myia/frontends/pytorch_functions.py

```diff
@@ -29,6 +29,27 @@
 # ############# THESE FUNCTIONS SHOULD BE IN ALPHABETICAL ORDER #############
 
 
+@core
+def conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1,
+           groups=1):
+    r"""Applies a Conv2d."""
+    # noqa: D202
+    """
+    # This is for later versions of pytorch that support other paddings?
+    if padding_mode != 'zeros':
+        raise Exception("'zeros' is the only padding_mode that is currently
+                        supported.")
+    #"""
+
+    if bias is not None:
+        ret = P.conv2d(input, weight, stride, padding, dilation, groups) \
+            + bias.reshape((1, bias.shape[0], 1, 1))
+    else:
+        ret = P.conv2d(input, weight, stride, padding, dilation, groups)
```

Comment:
It'd be a bit better to pull `P.conv2d(input, weight, stride, padding, dilation, groups)` out of the branches.